src/components/dataspaceDashboard/dataspaceSummary/DataspaceSummary.tsx

```diff
@@ -25,6 +26,9 @@ const fetchDataspacesList = async (): Promise<SelectOption[]> => {
   return await response.json()
 }
 
+const governanceLink =
```

Comment:
This one is weak and should be parametrized through environment variables.

Reply:
I'm on it 👍